extensions/ql-vscode/test/pure-tests/logging.test.ts

```diff
@@ -0,0 +1,156 @@
+import * as chai from 'chai';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import * as tmp from 'tmp';
+import 'mocha';
+import * as sinonChai from 'sinon-chai';
+import * as sinon from 'sinon';
+import * as pq from 'proxyquire';
+
+const proxyquire = pq.noPreserveCache().noCallThru();
+chai.should();
+chai.use(sinonChai);
+const expect = chai.expect;
+
+describe('OutputChannelLogger tests', () => {
+  let OutputChannelLogger;
+  const tempFolders: Record<string, tmp.DirResult> = {};
+  // let globalStoragePathTmp: tmp.DirResult;
```

Comment:
Are these commented lines supposed to be here?

Reply:
oops.